generators/app/templates/src/app/components/RadioGroup/index.js

```diff
@@ -1,60 +0,0 @@
-import React, { Component, createContext } from 'react';
-import PropTypes from 'prop-types';
-
-import RadioOption from './components/RadioOption';
-
-const { Provider, Consumer } = createContext();
-
-class RadioGroup extends Component {
-  static option = props => (
-    <Consumer>
-      {({ selectedValue, name, disabled, onChange }) => (
-        <RadioOption
-          {...props}
-          name={name}
-          onChange={onChange}
-          checked={selectedValue === props.value}
-          disabled={disabled || props.disabled}
-        />
-      )}
-    </Consumer>
-  );
```

Comment:
We should discuss about deleting this implementation.
The idea behind approach is to have the freedom to render any component as a child and not just options, and also style each option as needed.
It's probable that in most cases we don't need to, then we can map RadioGroup.option's inside a RadioGroup but i think it is useful to have that freedom. (Also, that is something we should replicate in other components)
Here #38 , in the pull request's description there is a brief description and an explanation about why i chose that.
Let me know what you think.
@pabloferro @damfinkel you too